tests/locations/test_package.py

```diff
@@ -1370,6 +1370,89 @@ def test_move_uncompressed_aip(self):
         assert pkg.current_location == dst_location
 
 
+@pytest.fixture
+@pytest.mark.django_db
+def space(tmp_path):
+    space_dir = tmp_path / "space"
+    space_dir.mkdir()
+
+    staging_dir = tmp_path / "staging"
+    staging_dir.mkdir()
+
+    space = models.Space.objects.create(
+        uuid=uuid.uuid4(),
+        access_protocol=models.Space.LOCAL_FILESYSTEM,
+        path=space_dir,
+        staging_path=staging_dir,
+    )
+    models.LocalFilesystem.objects.create(space=space)
+    return space
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def location(space):
+    aipstore = models.Location.objects.create(
+        uuid=uuid.uuid4(),
+        space=space,
+        relative_path="fs-aips",
+        purpose="AS",
+    )
+    models.Location.objects.create(
+        space=space, purpose=models.Location.STORAGE_SERVICE_INTERNAL, relative_path=""
+    )
```

Comment:
Why don't we move this model instance to its own fixture? It's not related to the AIPStorage location.